check_differences.bat

```diff
@@ -0,0 +1,122 @@
+@echo off
+
+rem On Windows, instead of using the diff command for SpeciminTestExecutor, we can use this
+rem command instead since diff is not included by default. This script outputs 0 if no differences
+rem are found, 1 if a difference in file structure or file content (all whitespace removed) is found
```

Comment:
Did you write this script yourself, or did you source it from somewhere? If the former, we need to add some tests for it (it's complex enough that I'm sure there are edge cases, and I'm not enough of a Windows batch file expert to be sure that this script is correct through inspection alone); if the latter, we should 1) give credit where credit is due, and 2) if possible, depend on wherever this script is sourced from directly, so that we'll receive bugfixes/updates if there are any.

Reply:
I wrote this script myself (with some Googling to help me with syntax). I can definitely add test cases to this, and definitely should. Thanks!